lib/iris/tests/test_concatenate.py

```diff
@@ -748,6 +730,49 @@ def test_concat_2x2d_aux_xy_bounds(self):
         self.assertEqual(len(result), 1)
         self.assertEqual(result[0].shape, (2, 4))
 
+    def test_fill_value_invariant_to_order__same_non_None(self):
```

Comment:
Yeah, test naming ... ugh ... you are right, I've don my thinking cap :cowboy_hat_face: